tests/basic/podmetric_test.go

```diff
@@ -263,3 +278,36 @@ func verifyPodHourWithError(actualPodHours, expectedPodHours, reasonableErrorPer
 
 	return nil
 }
+
+// updateStorageSpecRuntimeOpts updates the storageSpec's loggly callhome interval and
+// metering interval. Finally, restarts all PX pods and checks its condition.
+func updateStorageSpecRuntimeOpts(callhomeInterval string, meteringInterval string) {
```

Comment:
I am always voting for funcs to return things and then handle return in the test, but it seems there are many things that are going away from it, it just looks better when you go through test to see that we handling returned errors/ojects properly, rather than looking at every function that that doesn't return

Reply:
Thanks for noticing this @nikolaypopov, I've addressed this for Daniel since he's out.